spark/v4.0/spark/src/test/java/org/apache/iceberg/spark/source/TestPositionDeletesTable.java

```diff
@@ -1191,27 +1191,28 @@ public void testWritePartitionEvolutionDisallowed() throws Exception {
               .option(SparkReadOptions.FILE_OPEN_COST, Integer.MAX_VALUE)
               .load(posDeletesTableName);
       assertThat(scanDF.javaRDD().getNumPartitions()).isEqualTo(1);
+      assertThat(!Arrays.asList(scanDF.columns()).contains("partition"));
```

Comment:
Why do we need this assert?

Reply:
I can't remember why I added this line. I will remove this.